platform/8.0/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Jakarta Platform Spec 8.0"
+date: 2019-07-18T16:10:38-04:00 (use "date +%FT%T%z")
+summary: "Initial release of the Jakarta EE 8 Platform"
+---
+The Jakarta EE Platform defines a standard platform for hosting Jakarta EE applications.
+
+* [Jakarta EE Platform 8.0 Specification Document](./Platform-spec-8.0.pdf) (PDF)
+* [Jakarta EE Platform 8.0 Specification Document](./Platform-spec-8.0.html) (HTML)
+* Jakarta EE Platform 8.0 Javadoc - ??? (https://github.com/eclipse-ee4j/jakartaee-api)
```

Comment:
Just link to the apidocs directory that you'll include in a separate PR.

Reply:
Sure.  Done.